src/setup/rgw_start.py

```diff
@@ -27,8 +27,13 @@ class RgwStart:
     def start_rgw(conf: MappedConf, config_file, log_file, index: str = '1',):
         """Start rgw service independently."""
         try:
-            # TODO: To replace os.system with SimpleProcess calls
-            os.system(f"sh /opt/seagate/cortx/rgw/bin/rgw_service -i {index} -c {config_file} -l {log_file}")
-        except OSError as e:
+            cmd = f"/usr/bin/radosgw -f --name client.rgw-{index} -c {config_file} --no-mon-config &> {log_file}"
+            for _ in range(SERVICE_RETRIES):
+                _, stderr, rc = SimpleProcess(cmd).run()
+                if rc != 0:
```

Comment:
where is the break statement, when rc==0 ?

Reply:
There is no break statement in this case because the default nature of this loop is to stay in the SimpleProcess call. 
else if SimpleProcess fails for more than 5 times else will be executed and error will be logged in log file.

Reply:
Understood. We can add some code comments about the need for these retries, stating RGW start is retried to avoid failouts in case backend Motr or Hax is not reachable or available, temporarily.

Reply:
As per latest discussion, the retry logic is moved to RGW SAL layer.